pkg/trace/agent/run.go

```diff
@@ -116,6 +117,11 @@ func Run(ctx context.Context) {
 		defer os.Remove(flags.PIDFilePath)
 	}
 
+	// set core limits as soon as possible
```

Comment:
Please remove this comment.

Reply:
Removed everywhere since it's technically not "as soon as possible"